.changelog/20873.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:feature
+gateways: api-gateway can now use the new file-system-certificate configuration entry to reference a certificate and private key that are available in the local file system of the gateway.
```

Comment:
@jkirschner-hashicorp thoughts on this changelog entry?

Reply:
Alternative to consider

```suggestion
gateways: api-gateway can leverage listener TLS certificates available on the gateway's local filesystem by specifying the public certificate and private key path in the new file-system-certificate configuration entry
```